packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -267,6 +267,30 @@ export class DdSdkReactNativeConfiguration {
      */
     public bundleLogsWithTraces: boolean = DEFAULTS.bundleLogsWithTraces;
 
+    /**
+     * Enables tracking of non-fatal ANRs on Android.
```

Comment:
I'm wondering if we can call out these platform specific features in a better and more consistent way? Basically using the same language in both in the same place so that it's not missed...

Reply:
Maybe "This option is specific to Platform" as the second line and remove any other mention of platforms?